docs/data-export.md

```diff
@@ -29,7 +29,8 @@ You can also export this data using [the Scarf API](https://api-docs.scarf.sh/v2
 
 ## What Data is Exported
 
-The data export includes the following data fields:
+<a id="export-fields"></a>
```

Comment:
rather than having an empty anchor tag to use the id linking, should just use a heading here.

```suggestion
### Export Fields

```

Reply:
This assumes that the markdown hyperlinking to titles carries over with the library that builds the page?